sycl/include/CL/sycl/detail/kernel_desc.hpp

```diff
@@ -42,6 +42,8 @@ struct kernel_param_desc_t {
   // offset of the captured value of the parameter in the lambda or function
   // object
   int offset;
+  // bitfield to mark kernel parameters that will be omitted
+  unsigned char kernel_param_omitted : 1;
```

Comment:
If I understand correctly 1 means that this filed will take 1 bit only. How this works with accessor type where 4th argument is constructed using `param_omit_table[0] | (param_omit_table[1] << 1) | (param_omit_table[2] << 2) | (param_omit_table[3] << 3`"

Reply:
Thats right, this cannot be a bitfield.  
```suggestion
  unsigned char kernel_param_omitted;
```